crates/factor-key-value/src/host.rs

```diff
@@ -79,6 +80,7 @@ impl key_value::Host for KeyValueDispatch {}
 
 #[async_trait]
 impl key_value::HostStore for KeyValueDispatch {
+    #[instrument(name = "spin_key_value.open", skip(self), err(level = Level::INFO), fields(otel.kind = "client", kv.backend=self.manager.summary(&name).unwrap_or("unknown".to_string())))]
```

Comment:
I'm not sure that we should use `summary` here. It could have pretty high cardinality (and/or be overly verbose) for some stores like sqlite in a tempdir. Maybe we should add an e.g. `Store::type_name` method?

Reply:
Alternatively just omit `kv.backend` entirely and see if its actually a problem in practice :shrug:

Reply:
The more I think about this the less I think its actually a bad idea. Feel free to ignore.